src/store-pinia/userAuthStore.ts

```diff
@@ -0,0 +1,41 @@
+import {defineStore} from "pinia";
+import {reactive, ref} from "vue";
+import {ICredentialsInfo} from "@/services/auth.js";
+
+export const userAuthStore = defineStore('userAuth',
```

Comment:
Лучше константу стора начинать с use, ведь потом в коде использования будет вызов функции:
`const store = useUserAuthStore()`
```suggestion
export const useUserAuthStore= defineStore('userAuth',
```

Reply:
Коммит f7673c68